src/Helper/Product/Utility.php

```diff
@@ -49,6 +49,7 @@
 use Shopgate_Model_Catalog_Relation;
 use Shopgate_Model_Catalog_Validation;
 use Shopgate_Model_Catalog_Visibility;
+use function in_array;
```

Comment:
can you explain what this is for?
I looked into the usage of php core functions this way ind it seems to make sense if you want to use an alias.
This seems rather useless

Reply:
this is a performance optimization tweak based on: https://github.com/Roave/FunctionFQNReplacer

Reply:
I thought readability is the highes priority for our projects.
This is a line of code, which does not change the outcome and is thereby an abstraction.

But if you like it, keep it.

Reply:
Hm? Performance optimization is the outcome imho, readability of the use case block is sort of meh... Unless we disagree and then we may need a team meeting on it ^_^